homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -1103,6 +1161,63 @@ async def async_set_dry_off(self):
         )
 
 
+class XiaomiAirHumidifierMiot(XiaomiAirHumidifier):
+    """Representation of a Xiaomi Air Humidifier (MiOT protocol)."""
+
+    @property
+    def speed(self):
+        """Return the current speed."""
+        if self._state:
+            return AirhumidifierMiotOperationMode(self._state_attrs[ATTR_MODE]).name
+
+        return None
+
+    @property
+    def button_presed(self):
```

Comment:
```suggestion
    def button_pressed(self):
```
This is also unused?